src/Form/Form.php

```diff
@@ -211,7 +166,7 @@ protected function _buildValidator(Validator $validator)
      */
     public function buildValidator(Event $event, Validator $validator, $name)
     {
-        $this->_buildValidator($validator);
+        $this->setValidator($name, $validator);
```

Comment:
`setValidator()` is already called by `getValidator()`. The call to `_buildValidator()` was only for BC, so now this method can just be empty.

Reply:
Actually this method as well as `implementedEvents()` can be simply removed. For normal use case one you just use `validationDefault()` method to configure validations as done in tables.